partition-alloc/CMakeLists.txt

```diff
@@ -80,12 +84,11 @@ target_compile_options(partition-alloc
 		"-Wno-invalid-offsetof"
 		"-Wno-return-type"
 )
+target_compile_options(partition-alloc
+	PUBLIC
+		"-fno-exceptions"
```

Comment:
Why is this necessary?

Reply:
Partition alloc shim doesn't build with exceptions:

```c++
#if (defined(__GNUC__) && defined(__EXCEPTIONS)) || \
    (defined(_MSC_VER) && defined(_CPPUNWIND))
#error This code cannot be used when exceptions are turned on.
#endif
```